### DISCUSSION TITLE
CU-DU split: CU Failed to bind UDP socket to port 2152

### DISCUSSION THREAD

**Discussion by chonghua1** [srsran/srsRAN_Project#813] [General Help] [answered]
Hi community, 

I'm trying to deploy the CU and DU in two docker container. My setup is:
Open5GS runs in a container with ip 10.53.1.2; CU runs in container with ip 10.53.1.4; DU runs in container with ip 10.53.1.5

But I get following error:
```
srsran_cu         | --== srsRAN CU (commit ) ==--
srsran_cu         | 
srsran_cu         | N2: Connection to AMF on 10.53.1.2:38412 completed
srsran_cu         | F1-C: Listening for new connections on 10.53.1.4:38472...
srsran_cu         | Failed to bind UDP socket to 10.53.1.4:2152. Address already in use
srsran_cu         | srsRAN ERROR: Unable to allocate the required NG-U network resources
srsran_cu exited with code 1
```
It seems that the CU is using the same port (2152) for binding to both NG-U and F1-U. Can I change the port in config? Or any other methods to avoid the issue?

**Comment by chonghua1** [comment, marked as answer]
The issue has been solved through config multiple docker network. I'll close this discuss.

**Reply by Alan1321** [reply]
How did you fix it??
